perceval/__init__.py

```diff
@@ -52,6 +52,8 @@
 
 __version__ = PMetadata.version()
 
+from .providers import *
+from .converters import *
```

Comment:
We chose not to expose `convertors` to the top level of Perceval by default because most of the converter classes require 3rd parties that are optional (see the extra_requires in the setup.py file).
We'd like to stay that way.

The expected import statement for converter would then turn to:
`from perceval.converters import <Any>Converter`

On top of that, I think it's good practice to split imports of Perceval objects into sub-categories.

Reply:
I agree, I will revert the import of converters.